mt-batcher/mt_batcher.go

```diff
@@ -176,6 +176,12 @@ func NewMantleBatch(cfg Config) (*MantleBatch, error) {
 		HsmCreden:                 cfg.HsmCreden,
 		HsmFeeAPIName:             cfg.HsmFeeAPIName,
 		HsmFeeAddress:             cfg.HsmFeeAddress,
+		MinTimeoutRollupTxn:       cfg.MinTimeoutRollupTxn,
+		RollupTimeout:             cfg.RollupTimeout,
+	}
+	if cfg.MinTimeoutRollupTxn > cfg.RollUpMinTxn {
+		log.Error("new driver fail", "err", "MinTimeoutRollupTxn(%v)>RollUpMinTxn(%v)", cfg.MinTimeoutRollupTxn, cfg.RollUpMinTxn)
+		return nil, err
```

Comment:
The error information is incorrect, it is recommended to use errors.new() to return an error